CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.1.0
```

Comment:
do we want to align this version with the official release version of the go-algorand ?

Reply:
We should, this should already be 1.1.0 
and I think we should probably follow semantic versioning carefully across all products.

Reply:
I'm going to hold off of this until we have this conversation across all the repos. Seems like a larger decision to make.